Classes/Aggregate/ContentRenderingAggregate.php

```diff
@@ -90,7 +90,19 @@ protected function process()
 
         $this->addPlainTextFile(
             $this->typoScriptFilePath . 'constants.ts',
-            ''
+<<<EOS
+plugin.tx_mask {
+    view {
+        # cat=mask/file; type=string; label= Path to template root (FE)
+		templateRootPath = 
+
+		# cat=mask/file; type=string; label= Path to template partials (FE)
+		partialRootPath = 
+
+		# cat=mask/file; type=string; label= Path to template layouts (FE)
+		layoutRootPath = 
+	}
+EOS
```

Comment:
Please define each constant in one line. This enables opportunity for a new function check to ensure all constants used are properly set.
Furthermore there is a missing bracket at the end of the string.
Please remove the whitespace before the label